src/components/earn/PoolCard.tsx

```diff
@@ -151,17 +152,25 @@ export const PoolCard: React.FC<Props> = ({ compoundBotSummary }: Props) => {
   // })
 
   const [zapType, setZapType] = useState<ZapType | null>(null)
+  // TODO: show zap completed state rather than just collapsing the whole box
```

Comment:
should this be an issue?

Reply:
i thought it was just my OCD, it doesn't seem like a major issue (doesn't block functionality) but seeing the whole box collapse did take me by surprise. i'll create a non urgent issue for it :)